gcs/src/main/scala/org/broadinstitute/monster/storage/gcs/JsonHttpGcsApi.scala

```diff
@@ -453,7 +453,7 @@ private[gcs] class JsonHttpGcsApi(
         case OperationStatus.Done =>
           IO.pure(None)
       }
-      // Make sure there's at least one element in the result stream.
+        // Make sure there's at least one element in the result stream.
```

Comment:
Annoying change caused by bumping the scalafmt version via the plugins.

Might want to revert the version back from the RC... will think about it fresh tomorrow

Reply:
https://github.com/broadinstitute/monster-sbt-plugins/pull/2